packages/framework-provider-aws/src/library/subscription-adapter.ts

```diff
@@ -64,21 +64,6 @@ export async function fetchSubscriptions(
   return result.Items as Array<SubscriptionEnvelope>
 }
 
-export async function notifySubscription(
```

Comment:
Just removed this function form here, as it now belongs to the `connections-adatper.ts` file